.gitignore

```diff
@@ -20,3 +20,4 @@ out/
 jars/
 build/
 logs/
+null/
```

Comment:
For some reason it is automatically generated. Should have no side effect if adding this rule.